SHELLY_MJS.md

```diff
@@ -17,6 +17,13 @@ The script, when run, will fetch via REST api from a weather service the current
 cloud coverage is above or below certain percentage and respectively open or close window shades by calling a Shelly
 2.5 (Gen1) endpoint.
 
+precipitation-irrigation.js: Turn on/off watering based on precipitation in last 24 hours (based on AccuWeather data)
+===
+With this script you can stop or limit watering your garden according to the rain that has fallen in the last 24 hours.
```

Comment:
If there was precipitation in the past period (24h) skip an irrigation cycle. Data is retrieved from a public wather API service for the location.

Reply:
Done